src/Glpi/Form/Migration/FormMigration.php

```diff
@@ -0,0 +1,488 @@
+use Glpi\Message\MessageType;
+use Glpi\Migration\AbstractPluginMigration;
+use Plugin;
+
+final class FormMigration extends AbstractPluginMigration
+{
+    /**
+     * Version of FormCreator plugin required for the migration
+     */
+    private const FORMCREATOR_REQUIRED_VERSION = '2.13.9';
+
+    /**
+     * Retrieve the map of types to convert
+     *
+     * @return array
+     */
+    public function getTypesConvertMap(): array
+    {
+        return [
+            // TODO: We do not have a question of type "Actor",
+            // we have more specific types: "Assignee", "Requester" and "Observer"
+            'actor'       => QuestionTypeRequester::class,
+
+            'checkboxes'  => QuestionTypeCheckbox::class,
+            'date'        => QuestionTypeDateTime::class,
+            'datetime'    => QuestionTypeDateTime::class,
+            'dropdown'    => QuestionTypeItemDropdown::class,
+            'email'       => QuestionTypeEmail::class,
+            'file'        => QuestionTypeFile::class,
+            'float'       => QuestionTypeNumber::class,
+            'glpiselect'  => QuestionTypeItem::class,
+            'integer'     => QuestionTypeNumber::class,
+            'multiselect' => QuestionTypeDropdown::class,
+            'radios'      => QuestionTypeRadio::class,
+            'requesttype' => QuestionTypeRequestType::class,
+            'select'      => QuestionTypeDropdown::class,
+            'textarea'    => QuestionTypeLongText::class,
+            'text'        => QuestionTypeShortText::class,
+            'time'        => QuestionTypeDateTime::class,
+            'urgency'     => QuestionTypeUrgency::class,
+
+            // Description is replaced by a new block : Comment
+            'description' => null,
+
+            // TODO: Must be implemented
+            'fields'      => null,
+            'tag'         => null,
+
+            // TODO: This types are not supported by the new form system
+            // we need to define alternative ways to handle them
+            'hidden'      => null,
+            'hostname'    => null,
+            'ip'          => null,
+            'ldapselect'  => null,
+            'undefined'   => null,
+        ];
+    }
+
+    protected function validatePrerequisites(): bool
+    {
+        $this->result->addMessage(MessageType::Notice, 'Checking plugin version...');
+
+        $plugin = new Plugin();
+        if (!$plugin->getFromDBbyDir('formcreator')) {
+            $this->result->addMessage(MessageType::Error, 'Formcreator plugin is not installed.');
+            return false;
+        }
+
+        $is_version_ok = self::FORMCREATOR_REQUIRED_VERSION === $plugin->fields['version'];
+        if (!$is_version_ok) {
+            $this->result->addMessage(MessageType::Error, sprintf(
+                'Last Formcreator version (%s) is required to be able to continue.',
+                self::FORMCREATOR_REQUIRED_VERSION
+            ));
+            return false;
+        }
```

Comment:
Are we sure that this is necessary ? For instance, a 2.13.10 version has been released today. This check is already outdated.
IMHO, unless the corresponding fields types were recently udated, checking the DB fields presence is enough.

```suggestion
```